.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/DateParser.cs

```diff
@@ -52,13 +52,18 @@ public virtual DateTimeParseResult Parse(ExtractResult er, DateObject referenceD
                 Type = er.Type,
                 Data = er.Data,
                 Value = value,
-                TimexStr = value == null ? "" : ((DateTimeResolutionResult)value).Timex,
-                ResolutionStr = ""
+                TimexStr = value == null ? V : ((DateTimeResolutionResult)value).Timex,

```

Comment:
If the idea is to use a temp variable, let's give it a better name.

Reply:
But why is it needed? Especially only in this occurrence in this class.

Reply:
Not needed. Feedback applied.